\dropcap{S}tructure of the introduction
\newline
\begin{itemize}
\item Whole Genome Duplications background, major evolutionary force
\item the \textit{Paramecium} project, why \textit{Paramecium} is interesting, the aurelia complex
\item Here, focusing on the computational part, developing pipeline, showed that etc.
\end{itemize}

Since Ohno first hypothesized the influence of Whole-Genome Duplications (WGD), scientists kept showing that a broad number of species experienced WGD at least once: yeast, Drosophila, Vertebrates, Salmonids, Paramecium. WGD are evolutionary event when the genome of a given individual is duplicated, meaning that the whole genome is in two copies, duplicated pairs of genes are \textbf{paralogs}. WGD may be involved in many evolutive radiations as it creates a context of loosen selection. According to the Duplication-Degeneration-Complementation model,

To understand the consequences of WGDs we have been studying the \textit{Paramecium aurelia} complex. As one of the only free-living eukaryotes studied, other than yeasts, Paramecium is a very attractive model. The diversity of the \textit{Paramecium} ciliates is well studied. We focused on four species of \textit{Paramecium}: \textit{P. biaurelia}, \textit{P. sexaurelia}, \textit{P. tetraurelia} and \textit{P. caudatum} as an outgroup (see phylogenetic tree). The three \textit{aurelia} species underwent two rounds of WGDs, WGDX (... years ago) and WGDX (... years ago).

\textbf{Biological questions:} Do gene expression is linked, in \textit{P.}, with specific motifs? How are TFBS affected by WGD? Are they conserved among species, is this linked to expression level? Conserved among each species? Is there a bias of TFBS usage in certain species?