\section*{Future Directions}

We identified motifs using an automated pipeline (Fig.~\ref{fig:Pipeline}), comparing motifs identified using phylogenetic footprinting and statistical enrichment methods. We were able to dectect X motifs, associated with specific expression values and distance. Still, we plan to improve this pipeline in several ways.

After identifying motifs, we scan the entire \textit{Paramecium} genomes to find associated genes, however we only search for perfect matches. Thus, we do not take into account degenerate motifs. Thus we can miss genes that have slightly different motifs. Recently XXX showed that looking exact matches during motif identification may miss most of the signal. Indeed we have increase evidence that real motifs are most of the time degenerate and that Transcription Factor binding mostly depends on the general DNA context.

After identifying motifs, one's might want to group the motifs by similarity. We generally think about motifs as families, having specific sequences with some flexibility. For the moment, in our pipeline, we do not cluster motifs by "supposed" family, we consider each exact motifs as unique. However, several clustering methods exist to measure the diversity among motifs. The small sizes of motifs (about 6nt to 15nt) make these methods complicated to develop. Several methods have been implemented, but still, are not adapted for motifs and do not output families of motifs. 

Another way to think about the problem can be to cluster motifs according to features one can measure on them. For example, one can compute several statistical indexes on motifs to compare. Thus we do not need to implement a new algorithm to cluster motifs as other classical methods such as principal components analysis, hierarchal clustering or \textit{k}-means clustering can be applied. Still, these methods rely on the indexes you choose to measure, the motifs you are studying have to have well-defined clusters using those variables.

After identifying motifs should relate presence/absence of motifs to duplicated genes fate.

Motifs detection should take phylogeny into account for comparative analysis. Not the same value.

Need to improve pipeline for degenerate motifs, for the moment, extract only exact motif in the genome. (Ref. needed)

Need to measure diversity among detected motifs -> clustering tools and suppress rendundancy

Implement other motif finding tools to validate results.
