% Template for PLoS
% Version 1.0 January 2009
%
% To compile to pdf, run:
% latex plos.template
% bibtex plos.template
% latex plos.template
% latex plos.template
% dvipdf plos.template

\documentclass[10pt]{article}

% amsmath package, useful for mathematical formulas
\usepackage{amsmath}
% amssymb package, useful for mathematical symbols
\usepackage{amssymb}

% graphicx package, useful for including eps and pdf graphics
% include graphics with the command \includegraphics
\usepackage{graphicx}

% cite package, to clean up citations in the main text. Do not remove.
%\usepackage{cite}
\usepackage[round,sort]{natbib}

\usepackage{color} 
\usepackage[table]{xcolor}

% Use doublespacing - comment out for single spacing
%\usepackage{setspace} 
%\doublespacing


% Text layout
\topmargin 0.0cm
\oddsidemargin 0.5cm
\evensidemargin 0.5cm
\textwidth 16cm 
\textheight 21cm

% Bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,justification=raggedright]{caption}

% Use the PLoS provided bibtex style
%\bibliographystyle{plos2009}
\bibliographystyle{abbrvnat}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother


% Leave date blank
\date{}

\pagestyle{myheadings}

%% ** Colors **
\definecolor{linkcol}{RGB}{0,0,238}
\definecolor{light-gray}{gray}{0.85}

% \usepackage[english]{babel}
\usepackage[utf8]{inputenc}
\usepackage{hyperref}
\usepackage[hypcap]{caption}

%% For hyperref
\hypersetup{
    breaklinks,
    baseurl       = http://,
    colorlinks    = true,
    linkcolor     = linkcol,
    citecolor     = linkcol,
    pdfborder     = 0 0 0
}


%% END MACROS SECTION

\begin{document}

% Title must be 150 characters or less
\begin{flushleft}
{\Large
\textbf{Phylogenetic footprinting, a pipeline for motif identification in \textit{Paramecium}}
}
% Insert Author names, affiliations and corresponding author email.
\\
Matthias Grenié$^{1}$, 
Jean-François Goût$^{2}$, 
Michael Lynch$^{2}$
\\
\bf{1} Départment de Biologie, École Normale Supérieure de Lyon, Lyon, France
\\
\bf{2} Biology Department, Indiana University, IN, United States of America
\\
\end{flushleft}

% Please keep the abstract between 250 and 300 words
\section*{Abstract}


\input{Introduction.tex}

\input{Materials.tex}

\input{Results.tex}

\input{Perspectives.tex}

% Do NOT remove this, even if you are not including acknowledgments
\section*{Acknowledgments}

I would to thank my advisor Jean-François Goût for his patient and constant support, Michael Lynch for having me in his lab. More broadly, I would like to thank the whole Lynch lab team, for great scientific and non-scientific discussions.

\section*{Figures and Tables}

\begin{figure}[!ht]
\begin{center}
\includegraphics[scale=2.2]{Figures/TreeOfEuk.png}
\end{center}
\caption{
{\bf Phylogenetic tree of eukaryotes phylas.} The \textit{Ciliates} group, underlined in red in the figure, inside the Alveolates among the Chromoalveloates, it contains the \textit{Paramecium} genus. While the Animals (Metazoa) belong to the Opisthokonts group. From~\citep{keeling_tree_2005}
} 
\label{fig:TreeOfEuk}
\end{figure}


\begin{figure}[!ht]
\begin{center}
\includegraphics[scale=0.7]{Figures/SpeciesGene.png}
\end{center}
\caption{
{\bf Phylogenetic trees of \textit{Paramecium} and according gene tree.} A: \textit{Paramecium} species tree with WGD positions, \textit{caudatum} is the more basal species and we used it here as an outgroup, while the \textit{aurelia} cluster tightly together (Adapted from~\citealt{mcgrath_insights_2014}); B: An example of a family gene tree. With a single \textit{caudatum} gene and four genes for each \textit{aurelia} species, this is the maximum gene family size possible. In some families, certain genes of particular lineages may have been lost, the average size is 6.5, meaning that most of families actually lost some of the duplicate genes. \textit{Legend:} blue circle, intermediary WGD; orange circle, more recent WGD; as dated by~\citep{aury_global_2006}. (Adapted from \citealt{mcgrath_insights_2014}).
}
\label{fig:DuplicationTree}
\end{figure}


\begin{figure}[!ht]
\begin{center}
\includegraphics{Figures/Pipeline.png}
\end{center}
\caption{
{\bf Flow chart of the whole pipeline.}  From the genome assembly and annotation of the four species, we used orthology and paralogy groups from \citep{mcgrath_insights_2014}. For each of these families we extracted the coding sequences (CDS) as well as upstream regions from the start codon. We built a maximum likelihood phylogenetic tree using PhyML on pre-aligned CDS with TranslatorX \citep{abascal_translatorx:_2010}. On the one hand we computed the first fifth motifs of size of at least 4 nt using MEME on upstream regions \citep{bailey_meme:_2006}, while on the other hand using both the tree and the upstream sequences were used to detect motifs with a phylogenetic footprinting approach using BigFoot \citep{satija_bigfoot:_2009}. We then retained only conserved motifs between MEME and BigFoot.
}
\label{fig:Pipeline}
\end{figure}

\begin{figure}[!ht]
\begin{center}
\includegraphics{Figures/DegenerateMotif.png}
\end{center}
\caption{
{\bf Degenerate Motif Example.}  This is a binding site from \citealt{whitfield_functional_2012}, in human promoters. Some positions are clearly conserved, like position 5 with T, while others are highly variable (positions 1 and 4). Transcription Factor Binding Sites are very conserved on some positions while highly variable on others.
}
\label{fig:DegenerateMotif}
\end{figure}

\begin{figure}[!ht]
\begin{center}
\includegraphics[scale=0.6]{Figures/ScoresPlot.png}
\end{center}
\caption{
{\bf Motif detection from BigFoot example.} BigFoot proceeds to phylogenetic prediction using an alignment technique, refer to~\citep{satija_bigfoot:_2009} for more informations. For each position in the alignment, BigFoot assigns an alignment confidence score and phylogenetic footprinting,\textit{i.e.} motif prediction, confidence score. In our analysis we used a detection technique to allow gaps in both the prediction and the alignment score. Curves: Red solid: Prediction score, Red dashed: Prediction threshold used, Blue solid: Alignment score, Blue dashed: Alignment threshold. Gray boxes: potential motifs detected in the sequence.
}
\label{fig:Scores}
\end{figure}

\begin{figure}[!ht]
\begin{center}
\includegraphics[scale=0.5]{Figures/PropFigure.pdf}
\end{center}
\caption{
{\bf Ratio of genes with detected common motifs in all the species.} Legend: A, Proportion of genes containing the motif by species ; B, separated by motifs and by species ; C, grouped by motif. bi: \textit{P. biaurelia}; ca: \textit{P. caudatum}; sex: \textit{P. sexaurelia}; tet: \textit{P.tetraurelia}. \textit{P. caudatum} seems, for all motifs, to have less genes containing them then the other species. While two motifs seems importantly lower than others: TAAATCT and TTAATATT, but TTAATATT and TTAATTA seem also lower than other motifs. For more details on the ratios, see~\autoref{tab:Motifs}
}
\label{fig:PropFigure}
\end{figure}

\input{Figures/MotifsTable.tex}

%\section*{References}
% The bibtex filename
\clearpage
\bibliography{references.bib}

\end{document}

