% Template for PLoS
% Version 1.0 January 2009
%
% To compile to pdf, run:
% latex plos.template
% bibtex plos.template
% latex plos.template
% latex plos.template
% dvipdf plos.template

\documentclass[10pt]{article}

% amsmath package, useful for mathematical formulas
\usepackage{amsmath}
% amssymb package, useful for mathematical symbols
\usepackage{amssymb}

% graphicx package, useful for including eps and pdf graphics
% include graphics with the command \includegraphics
\usepackage{graphicx}

% cite package, to clean up citations in the main text. Do not remove.
\usepackage{cite}

\usepackage{color} 

% Use doublespacing - comment out for single spacing
%\usepackage{setspace} 
%\doublespacing


% Text layout
\topmargin 0.0cm
\oddsidemargin 0.5cm
\evensidemargin 0.5cm
\textwidth 16cm 
\textheight 21cm

% Bold the 'Figure #' in the caption and separate it with a period
% Captions will be left justified
\usepackage[labelfont=bf,labelsep=period,justification=raggedright]{caption}

% Use the PLoS provided bibtex style
\bibliographystyle{plos2009}

% Remove brackets from numbering in List of References
\makeatletter
\renewcommand{\@biblabel}[1]{\quad#1.}
\makeatother


% Leave date blank
\date{}

\pagestyle{myheadings}
%% ** EDIT HERE **

% \usepackage[english]{babel}
\usepackage[utf8]{inputenc}

%% ** EDIT HERE **
%% PLEASE INCLUDE ALL MACROS BELOW

%% END MACROS SECTION

\begin{document}

% Title must be 150 characters or less
\begin{flushleft}
{\Large
\textbf{Title}
}
% Insert Author names, affiliations and corresponding author email.
\\
Matthias Grenié$^{1}$, 
Jean-François Goût$^{2}$, 
Michael Lynch$^{2}$
\\
\bf{1} Départment de Biologie, École Normale Supérieure de Lyon, Lyon, France
\\
\bf{2} Biology Department, Indiana University, IN, United States of America
\\
\end{flushleft}

% Please keep the abstract between 250 and 300 words
\section*{Abstract}

% Please keep the Author Summary between 150 and 200 words
% Use first person. PLoS ONE authors please skip this step. 
% Author Summary not valid for PLoS ONE submissions.  
\section*{Author Summary}

\input{Introduction.tex}

\input{Materials.tex}

\input{Results.tex}

\input{Perspectives.tex}

% Do NOT remove this, even if you are not including acknowledgments
\section*{Acknowledgments}

I would to thank my advisor Jean-François Goût for his patient and constant support, Michael Lynch for having me in his lab. More broadly, I would like to thank the whole Lynch lab team, for great scientific and non-scientific discussions.

\section*{References}
% The bibtex filename
\bibliography{template}

\section*{Figure Legends}

\begin{figure}[!ht]
\begin{center}
\includegraphics{Figures/TreeOfEuk.png}
\end{center}
\caption{
{\bf Phylogenetic tree of eukaryotes phylas.} The \textit{Ciliates} group, underlined in red in the figure, inside the Alveolates among the Chromoalveloates, it contains the \textit{Paramecium} genus. While the Animals (Metazoa) belong to the Opisthokonts group. From (Ref. needed)
} 
\label{TreeOfEuk}
\end{figure}


\begin{figure}[!ht]
\begin{center}
\includegraphics{Figures/Pipeline.png}
\end{center}
\caption{
{\bf Flow chart of the whole pipeline.}  From the genome assembly and annotation of the four species, we used orthology and paralogy groups from (Ref. needed). For each of these families we extracted the coding sequences (CDS) as well as upstream regions from the start codon. We built a maximum likelihood phylogenetic tree using PhyML on pre-aligned CDS with TranslatorX. On the one hand we computed the first fifth motifs of size of at least 4 nt using MEME on upstream regions, while on the other hand using both the tree and the upstream sequences were used to detect motifs with a phylogenetic footprinting approach using BigFoot. We then retained only conserved motifs between MEME and BigFoot.
}
\label{Pipeline}
\end{figure}

\begin{figure}[!ht]
\begin{center}
\includegraphics{Figures/DuplicationPatterns.png}
\end{center}
\caption{
{\bf One possible gene tree.} An example of a family gene tree. A single \textit{caudatum} gene and four genes for each \textit{aurelia} species. \textit{Legend:} blue circle, intermediary WGD; orange circle, more recent WGD; as dated by (Ref. needed). Adapted from (Ref. needed)
}
\label{DuplicationTree}
\end{figure}


\section*{Tables}
%\begin{table}[!ht]
%\caption{
%\bf{Table title}}
%\begin{tabular}{|c|c|c|}
%table information
%\end{tabular}
%\begin{flushleft}Table caption
%\end{flushleft}
%\label{tab:label}
% \end{table}

\end{document}

