\section*{Results}
\label{sec:Results}

From our orthologs and paralogs gene familes we had 5751 families, only 5008 gene families had at least 4 genes. We ran our pipeline on these 5008 families, to detect conserved motifs among upstream sequences of \textit{Paramecium} genomes. The pipeline was set to detect motifs, in upstream regions ranging between 15nt and 250nt (see~\nameref{sec:Methods}) , of at least 6 nucleotides with an eventual gap in conservation (see~\autoref{fig:DegenerateMotif}) for more biological relevance.

Using BigFoot we identified 811 different motifs in 608 families. Comparing these motifs with MEME ones for partial match, only 117 unique motifs were retained. For each of these, we extracted the name of all genes containing the motif, in each species. To focus only on conserved motifs, we conserved only motifs matching in all the four species. Only 10 motifs matched these conditions. We then computed the ratio of genes containing the motif over the total number of genes (results are summed up in~\autoref{tab:Motifs}).